<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>548</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>548</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="dance_horizontalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>521</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="dance_prepare_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QPushButton" name="DCOff1_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect3_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOn3_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC On</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dance_comboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="robotName1_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ERROR1</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileName2_label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>18</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>file:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileName3_comboBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect1_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOff3_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Off</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileName2_comboBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="robotName2_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ERROR2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOff2_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect1_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="dance_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>34</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Dance:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect2_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basicPosture1_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic Posture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basicPosture2_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic Posture</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileName1_comboBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="fileName1_label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>18</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>file:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOn2_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC On</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileName3_label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>18</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>file:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOn1_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect3_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="robotName3_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ERROR3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basicPosture3_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic Posture</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="music_checkBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>49</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect2_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectAll_pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectAll_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basicPostureAll_pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic Posture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOnAll_pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DCOffAll_pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Off</string>
     </property>
    </widget>
    <widget class="QLabel" name="robotNameAll_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>135</y>
       <width>20</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="robotControls_line">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>135</y>
      <width>515</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>dance_prepare_widget</zorder>
   <zorder>dance_horizontalSlider</zorder>
   <zorder>start_pushButton</zorder>
   <zorder>pause_pushButton</zorder>
   <zorder>stop_pushButton</zorder>
   <zorder>robotControls_line</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
